assets/airports/lszh.json

```diff
@@ -0,0 +1,1812 @@
+                "LSZH16": ["LAMAX", ["@AMIKI", "A70+"]],
+                "LSZH28": [["ZH264", "A100+"], ["ZH265", "A80+"], "ZH266", "EKRIT", "ZH268", "ZH269", "ZH270", ["ZH280", "A70+"], "ZH281", ["RAMEM", "A50+"]],
+                "LSZH32": ["LAMAX", ["@AMIKI", "A70+"]],
+                "LSZH34": [["ZH363", "A100+"], "ZH364", ["ZH365", "A80+"], ["ZH366", "A70+"], "ZH367", "ZH368", "ZH369", "ZH370", ["ZH340", "S200-"], ["UTIXO", "A60+"], ["MILNI", "A50+"]]
+            },
+            "draw": [
+                ["RILAX", "LAMAX", "AMIKI"]
+            ]
+        },
+        "TRA1Z": {
+            "icao": "TRA1Z",
+            "name": "Trasadingen One Zulu",
+            "entryPoints": {
+                "TRA": ["TRA"]
+            },
+            "body": [["@AMIKI", "A70+"]],
+            "rwy": {
+                "LSZH10": [],
+                "LSZH14": [],
+                "LSZH16": [],
+                "LSZH28": ["SONGI", "ZH266", "EKRIT", "ZH268", "ZH269", "ZH270", ["ZH280", "A70+"], "ZH281", ["RAMEM", "A50+"]],
+                "LSZH32": [],
+                "LSZH34": ["SONGI", ["ZH365", "A80+"], ["ZH366", "A70+"], "ZH367", "ZH368", "ZH369", "ZH370", ["ZH340", "S200-"], ["UTIXO", "A60+"], ["MILNI", "A50+"]]
+            },
+            "draw": [
+                ["TRA", "AMIKI"]
+            ]
+        },
+        "URNAS2Z": {
+            "icao": "URNAS2Z",
+            "name": "Urnas Two Zulu",
+            "entryPoints": {
+                "URNAS": [["URNAS", "A140+"]]
+            },
+            "body": [["ALBIX", "A100+"], ["ZH650", "A80+"], "ZH676", ["@GIPOL", "A70+"]],
+            "rwy": {
+                "LSZH10": [],
+                "LSZH14": [],
+                "LSZH16": [],
+                "LSZH28": ["ZH203", "ZH204", "ZH205", "ZH206", "ZH207", "ZH208", "ZH209", "ZH210", ["ZH280", "A70+"], "ZH281", ["RAMEM", "A50+"]],
+                "LSZH32": [],
+                "LSZH34": [["ZH305", "A70+"], "ZH306", "ZH307", "ZH308", "ZH309", "ZH310", ["ZH340", "S200-"], ["UTIXO", "A60+"], ["MILNI", "A50+"]]
+            },
+            "draw": [
+                ["URNAS", "ALBIX", "ZH650", "ZH676", "GIPOL"]
+            ]
+        },
+        "WIL2Z": {
+            "icao": "WIL2Z",
+            "name": "Willisau Two Zulu",
+            "entryPoints": {
+                "WIL": ["WIL"]
+            },
+            "body": [["@GIPOL", "A70+"]],
+            "rwy": {
+                "LSZH10": [],
+                "LSZH14": [],
+                "LSZH16": [],
+                "LSZH28": ["ZH203", "ZH204", "ZH205", "ZH206", "ZH207", "ZH208", "ZH209", "ZH210", ["ZH280", "A70+"], "ZH281", ["RAMEM", "A50+"]],
+                "LSZH32": [],
+                "LSZH34": [["ZH305", "A70+"], "ZH306", "ZH307", "ZH308", "ZH309", "ZH310", ["ZH340", "S200-"], ["UTIXO", "A60+"], ["MILNI", "A50+"]]
+            },
+            "draw": [
+                ["WIL", "GIPOL"]
+            ]
+        }
+    },
+    "spawnPatterns": [

```

Comment:
Would it be better to use airways here? For examples of airway usage, you can have a look at feature/199 or `egcc.json`.